schedule/yast/autoyast_salt_formulas.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: autoyast_salt_formulas
+description: |
+  Test installation using AY plus salt formulas.
+  Test formulas used is a testing one from https://w3.suse.de/~igonzalezsosa/salt/
+schedule:
+  - autoyast/prepare_profile
+  - installation/isosize
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
```

Comment:
what do you think here about to clone ay profile and then then do some simple validation with more relevant tags?

Reply:
might be useful. i would keep this simple tho and i would create another ticket to break it down as a task.

Reply:
we don't usually create tickets for these kind of thing. I recommend you to do it in the scope of this ticket, at least to schedule the clone profile test module, as it is part of any manual testing of any feature. Validation of the cloned profile would be nice to have and quick to do, but I leave that to you.
In this case if you don't want to do it now, as discussed offline, if you don't mind you can describe to schedule those 2 modules in the ticket about using salt Forms that you will create as a follow-up.

Reply:
ticket was created.